news.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: allposts
+title: News
+landing-title: 'News'
+nav-menu: true
+description: null
+image: null
+author: null
+show_tile: false
+---
+
+# News
```

Comment:
News should show a list of new items so that it is much easier to navigate. This is what the current website has.

Reply:
I agree that a list of items would be easier to navigate. I propose that we look at how to do this when we have more content.

Reply:
See bc56ed61b7c2ebe0882ff7bfa6142d5caf871b88

Reply:
@mchonofsky This isn't an appropriate fixed. The list is not that easy to find and when you click on news from the "hamburger menu" it still takes you a huge page that just dumps all the posts on it.

What I wanted was for when you click on "News" to be taken to a list of articles where click on an item in the list takes you to the full article. This is what the existing site does and you can see the templates for this at

https://github.com/UnionOfUKUnicyclists/unionofukunicyclists.github.io/blob/8374c8e1365b28c5813a721b56ea07191c21b3b9/_includes/blog_list.html

and

https://github.com/UnionOfUKUnicyclists/unionofukunicyclists.github.io/blob/8374c8e1365b28c5813a721b56ea07191c21b3b9/_includes/blog_post.html

You've also removed the `permalink: '/blog/:year/:month/:day/:title/'` entry from `_config.yml`. Please put it back (probably replace `blog` with `news`). Otherwise the news posts won't have a sensible permalink.

Reply:
@mchonofsky If you want to punt this and fix it later, then that's fine but please create a new issue in the issue tracker for this if you decide to do this.

Reply:
I'll get most of it into a new commit in a short while. It wasn't a removal of the permalink statement - it just wasn't in the layout templates I was working from. I'll add that back in. I'll punt on the blog post list issue for now but sort something out eventually (via an issue in the tracker, of course!)